### PR TITLE
Initial commit SigmaConsoCR connector

### PR DESCRIPTION
**Pull request by tve-sigmaconso**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] Independent Publisher Connectors Only: Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] ndependent Publisher Connectors Only: Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/SigmaConsoCR/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+      "title": "Sigma Conso CR",
+      "description": "Sigma Conso Consolidation & Reporting Connector provides a set of APIs to interact with your Consolidation & Reporting instances. Easily transfer data, trigger actions and automate workflows in Sigma Conso. Go beyond the limits of traditional CPM solutions.",
+      "version": "1.0",
+      "contact": {
+        "name": "Sigma Conso",
+        "url": "https://www.sigmaconso.com/",
+        "email": "support@sigmaconso.com"
+      }
+    },
+    "host": "x",
```
Comment (sriyen-msft) [inline certified-connectors/SigmaConsoCR/apiDefinition.swagger.json:13]: @tve-sigmaconso Providing `x` would be a syntax error in the build system, Can you please update it with `"yourhost.yourdomain.com"`

File: certified-connectors/SigmaConsoCR/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+                "type": "object",
+                "properties": {
+                  "Process": {
+                    "type": "string",
+                    "description": "Process"
+                  },
+                  "Action": {
+                    "type": "string",
+                    "description": "Action",
+                    "title": ""
```
Comment (sriyen-msft) [inline certified-connectors/SigmaConsoCR/apiDefinition.swagger.json:63]: @tve-sigmaconso Can you please let us know the reason to provide empty string for title properties? If they are not required, you can remove them.

File: certified-connectors/SigmaConsoCR/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+                  },
+                  "CustomerCode": {
+                    "type": "string",
+                    "description": "CustomerCode",
```
Comment (sriyen-msft) [inline certified-connectors/SigmaConsoCR/apiDefinition.swagger.json:67]: @tve-sigmaconso The description values should be free from grammatical errors, Can it be `"Customer Code"` ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/SigmaConsoCR/apiDefinition.swagger.json (hunk 1 above)
Comment (tve-sigmaconso) [reply certified-connectors/SigmaConsoCR/apiDefinition.swagger.json]: This is fixed in commit 47bbf27e.

File: certified-connectors/SigmaConsoCR/apiDefinition.swagger.json (hunk 3 above)
Comment (tve-sigmaconso) [reply certified-connectors/SigmaConsoCR/apiDefinition.swagger.json]: This is fixed in commit 47bbf27e

File: certified-connectors/SigmaConsoCR/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+                "properties": {
+                  "Message": {
+                    "type": "string",
+                    "description": "Message"
+                  }
+                }
+              }
+            }
+          },
+          "summary": "Consolidation",
```
[the remark this answers is not in the document]
Comment (tve-sigmaconso) [reply certified-connectors/SigmaConsoCR/apiDefinition.swagger.json:99]: This is fixed in commit 47bbf27e

File: certified-connectors/SigmaConsoCR/apiDefinition.swagger.json (hunk 2 above)
Comment (tve-sigmaconso) [reply certified-connectors/SigmaConsoCR/apiDefinition.swagger.json]: This is fixed in commit 47bbf27e

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**